### PR TITLE
setup.py: cleanup

### PR DESCRIPTION
**Pull request by toabctl**
It is not used in the code.

### CODE REVIEW

**Comment by toabctl** [comment]
> The change is more than what's described in the PR title, and apparently breaks the build

that's why there are 2 commits (which describe each change). But I adjusted the PR title

**Comment by yurishkuro** [comment]
we squash the commits on merge. The build still fails.

**Comment by toabctl** [comment]
> we squash the commits on merge.

Oh. So you loose all the good information from commit messages. So why do you want to squash the commits? What's the reason for that?

**Comment by toabctl** [comment]
> The build still fails.

Fixed my mistakes. But not it runs into https://github.com/opentracing/opentracing-python/issues/115

**Comment by yurishkuro** [comment]
> So why do you want to squash the commits? What's the reason for that?

because in most cases those commits are just noise, with bad commit messages.

> But not it runs into #115

you can pin tornado < 6 for now.

**Comment by toabctl** [comment]
> because in most cases those commits are just noise, with bad commit messages.

Please don't do this with this commit. I think the messages **are** useful. Otherwise I'm happy to adjust the commit messages (it's githubs fault that you can't review commit messages...)
> 
> > But not it runs into #115
> 
> you can pin tornado < 6 for now.

Done

**Comment by yurishkuro** [comment]
GitHub keeps all the commit messages in the final commit. I would rather have the PR title reflect the change and be the top-level commit title.

**Comment by toabctl** [comment]
> GitHub keeps all the commit messages in the final commit. I would rather have the PR title reflect the change and be the top-level commit title.

Hm. Why? What's wrong with having 3 commits where every commit describes what and why? In the case the commits are not noise and not bad (I think). And commit messages (and small commits) are important to understand **why** something changed and also for backporting stuff to older releases (that's what distro usually need after some time).

**Comment by yurishkuro** [comment]
> Hm. Why? 

1. this is our policy
2. in most cases people are not diligent about good commit messages. It is a lot more friction for project maintainers to keep reminding everyone "please squash" or "please follow guidelines on commit messages" than to have a "squash and merge" policy.

**Comment by evrardjp** [comment]
On the other hand, squashing those commits into one sounds like a bad idea. They are all separate, can be evaluated separately.

I guess this can be done as 3 PRs instead, but do you really want to evaluate 3 PRs to cleanup the cruft?

File: setup.py [hunk 1]
```diff
@@ -35,10 +34,13 @@
             'pytest-mock',
             'Sphinx',
             'sphinx_rtd_theme',
-
             'six>=1.10.0,<2.0',
+        ],
+        'tornado': [
+            'tornado<6',
```
Comment (evrardjp) [inline setup.py]: can this be based on python version, instead of reducing the opportunities with python3?

**Comment by toabctl** [comment]
I don't plan to work on this. Closing.

**Pull request closed without merging**